explainer.md

```diff
@@ -1,624 +1,396 @@
+* Entry/Exit Transitions: A UI element animates as it exits or enters the screen. This [issue](https://github.com/WICG/shared-element-transitions/issues/37) shows an example.
 
 # Design
 
-Let's take the example below which shows how the API can be used by a developer
-to animate the background and a shared element on a same origin navigation
-(MPA). The SPA equivalent of this case is one where the old document is mutated
-into the new document via DOM APIs. See [API Extensions](#api-extensions) for
-more code examples.
+The goal is to provide a mechanism and API which will allow simple transitions like above to be specified in CSS, building on CSS animations, but also allow for more complex transition to be performed via JavaScript, building on the Web Animations API.
 
-Note that the API shape below is tentative and used to explain the core feature
-design.
+This section covers the concepts and mechanisms, while a later section looks at possible API shapes.
 
-### Old Document
+Performing a transition from Page-A to Page-B requires parts of both to be on screen at the same time, potentially moving independently. This is currently impossible in a cross-document navigation, but it's still hard in an SPA (single page app) navigation. You need to make sure that the outgoing state can't receive additional interactions, and ensure the presence of both states doesn't create a confusing experience for those using accessibility technology.
 
-```html
-<html>
-<head>
-  <style>
-    body {
-      background-color: blue;
-    }
-    .animated {
-      position: relative;
-      top: 300px;
-      width: 100px;
-      height: 100px;
-      background-color: red;
-    }
-  </style>
-</head>
-<body>
-  <div class="animated" sharedid="header-id" id="header">Shared Element</div>
-  <a href="new-document.html">Click Me</a>
-</body>
-</html>
-```
+The aim of this design is to allow for representations of both Page-A and Page-B to exist at the same time, without the usability, accessibility, or memory concerns of having both complete DOM trees alive.
 
-### New Document
+Here's the example that will be used to explain the design:
 
-```html
-<html>
-<head>
-  <style>
-    body {
-      background-color: lightblue;
-    }
-    .animated {
-      position: relative;
-      top: 100px;
-      width: 200px;
-      height: 200px;
-      background-color: green;
-    }
-
-    ::shared-container(header-id) {
-      animation: ::shared-container-header-id 1s ease-in;
-    }
-  </style>
-</head>
-<body>
-  <div class="animated" sharedid="header-id" id="header">Shared Element</div>
-</body>
-</html>
+<img alt="Page-A and Page-B" src="media/pages.png?raw=true">
+
+The transition between Page-A and Page-B can be a full cross-document navigation between two same-origin pages, or it can be an SPA navigation. The main mechanism and concepts are the same between the two.
```

Comment:
This seems repetitive with line 26?

Reply:
I'll try to reword it. I'm trying to say "these concepts apply to both MPA and SPA, although the API may differ"